pydevd_attach_to_process/attach_script.py

```diff
@@ -81,6 +81,10 @@ def fix_main_thread_id(on_warn=lambda msg: None, on_exception=lambda msg: None,
     import sys
     import threading
 
+    # This no longer works in Py 3.13
```

Comment:
Not sure about this change either. The internals used here aren't all available in 3.13. Disabling it caused more tests to pass though.

Reply:
It seems that the issue for this hack is now fixed, so, this isn't needed anyways anymore in Python 3.13.